plugins/optimization-detective/storage/data.php

```diff
@@ -179,6 +179,50 @@ function od_verify_url_metrics_storage_nonce( string $nonce, string $slug, strin
 	return (bool) wp_verify_nonce( $nonce, "store_url_metrics:$slug:$url" );
 }
 
+/**
+ * Gets the minimum allowed viewport aspect ratio for URL metrics.
+ *
+ * @since n.e.x.t
+ * @access private
+ *
+ * @return float Minimum viewport aspect ratio for URL metrics.
+ */
+function od_get_minimum_viewport_aspect_ratio(): float {
+	/**
+	 * Filters the minimum allowed viewport aspect ratio for URL metrics.
+	 *
+	 * The 0.4 value is intended to accommodate the phone with the greatest known aspect
+	 * ratio at 21:9 when rotated 90 degrees to 9:21 (0.429).
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param float $minimum_viewport_aspect_ratio Minimum viewport aspect ratio.
+	 */
+	return (float) apply_filters( 'od_minimum_viewport_aspect_ratio', 0.4 );
+}
+
+/**
+ * Gets the maximum allowed viewport aspect ratio for URL metrics.
+ *
+ * @since n.e.x.t
+ * @access private
+ *
+ * @return float Maximum viewport aspect ratio for URL metrics.
+ */
+function od_get_maximum_viewport_aspect_ratio(): float {
+	/**
+	 * Filters the maximum allowed viewport aspect ratio for URL metrics.
+	 *
+	 * The 2.5 value is intended to accommodate the phone with the greatest known aspect
```

Comment:
```suggestion
	 * The 2.5 default value is intended to accommodate the phone with the greatest known aspect
```
added word "default" because if these docs get referenced elsewhere the value may not be visible so it won't be clear what this refers to